infrastructure/images.py

```
@@ -0,0 +1,47 @@
+import os
+from datetime import datetime
+from glob import glob
+import pulumi
+import pulumi_docker as docker
```

Ruff (0.11.9)

`pulumi_docker` imported but unused

Remove unused import: `pulumi_docker`

(F401)